integration/test/performance_tracker/transmitter/transmitter.go

```diff
@@ -0,0 +1,236 @@
+	Id string	`json:"Id"`
+	Label string	`json:Label`	
+	Messages string	`json:Messages`
+	StatusCode string	`json:StatusCode`
+	Timestamps []string	`json:Timestamps`
+	Values []float64	`json:Values`
+}
+
+/*
+InitializeTransmitterAPI
+Desc: Initializes the transmitter class
+Side effects: Creates a dynamodb table if it doesn't already exist
+*/
+func InitializeTransmitterAPI(DataBaseName string) * TransmitterAPI{
+	//setup aws session
+	session := session.Must(session.NewSessionWithOptions(session.Options{
+		SharedConfigState: session.SharedConfigEnable,
+	}))
+	transmitter := TransmitterAPI{
+		dynamoDbClient: dynamodb.new(session),
+		DataBaseName:	DataBaseName,
+	}
+	// check if the dynamo table exist if not create it
+	tableExist, err:= transmitter.TableExist()
+	if err !=nil{
+		return nil
+	}
+	if !tableExist{
+		fmt.Println("Table doesn't exist")
+		err := transmitter.CreateTable()
+		if err != nil{
+			return nil
+		}
+	}
+	fmt.Println("API ready")
+	return &transmitter
+
+}
+/*
+CreateTable()
+Desc: Will create a DynamoDB Table with given param. and config
+*/
+func (transmitter * TransmitterAPI) CreateTable() error{
+	input := &dynamodb.CreateTableInput{
+		AttributeDefinitions: []*dynamodb.AttributeDefinition{
+			{
+				AttributeName: aws.String("Hash"),
+				AttributeType: aws.String("S"),
+			},
+		},
+		KeySchema: []*dynamodb.KeySchemaElement{
+			{
+				AttributeName: aws.String("Hash"),
+				KeyType:       aws.String("HASH"),
+			},
+		},
+		ProvisionedThroughput: &dynamodb.ProvisionedThroughput{
+			ReadCapacityUnits:  aws.Int64(10),
+			WriteCapacityUnits: aws.Int64(10),
+		},
+		TableName: aws.String(transmitter.DataBaseName),
+	} // this is the config for the new table
+	
+	_, err := transmitter.dynamoDbClient.CreateTable(input)
+	if err != nil {
+		fmt.Printf("Error calling CreateTable: %s", err)
+		return err
+	}
+	
+	fmt.Println("Created the table", transmitter.DataBaseName)
+	return nil
+}
+/*
+AddItem() 
+Desc: Takes in a packet and
+will convert to dynamodb format  and upload to dynamodb table.
+Param: 
+	packet * map[string]interface{}:  is a map with data collection data
+Side effects: 
+	Adds an item to dynamodb table
+*/
+func (transmitter * TransmitterAPI) AddItem(packet map[string]interface{})(string,error){
+	item, err := dynamodbattribute.MarshalMap(packet)
+	if err != nil {
+		panic(err)
+	}
+	dbItem := &dynamodb.PutItemInput{
+		Item:      item,
+		TableName: aws.String(transmitter.DataBaseName),
+	}
+	_, err = transmitter.dynamoDbClient.PutItem(dbItem)
+	if err != nil {
+		fmt.Printf("Error adding item to table.  %v\n", err)
+	}
+	return fmt.Sprintf("%v",item),err
+}
+/*
+TableExist()
+Desc: Checks if the the table exist and returns the value
+//https://github.com/awsdocs/aws-doc-sdk-examples/blob/05a89da8c2f2e40781429a7c34cf2f2b9ae35f89/gov2/dynamodb/actions/table_basics.go
+*/
+func (transmitter * TransmitterAPI) TableExist() (bool,error){
```

Comment:
**Not Nit pick**: Provide the `context.Background()` as the top level and pass down to input function would be a greater idea since `context` is used for avoid SLE case and wasteful memory resources which we will handle

Reply:
Now that #497 is merged, to integrate with this, we can pass in a `context.Context` parameter that gets instantiated in the performance test.